### ISSUE TITLE
info manual

### ISSUE THREAD

**Issue by holomorph** [magnars/dash.el#119]
It would be nice if dash had an info manual.  Perhaps this could be achieved by:

* pandoc
* rewriting the skeleton in org, writing a new org generator and doing org -> info
* using some other format
* ???

**Comment by Fuco1** [comment]
I don't oppose the change to org in principle, it depends (for me) on the fact whether github can render org as well as markup.

Autogenerating org or md markup is on par in difficulty (= very simple) and the skeleton can be rewritten by hand easily as well.

**Comment by holomorph** [comment]
Played around and did the skeleton https://gist.github.com/holomorph/ed2ef265d3b9eb2dbf15

**Comment by magnars** [comment]
Sounds like a fine idea to me. However, last time I tried it, github really did a horrible job of rendering org. That might have changed now, tho. Do a test if you can make it look good before dumping too much time into generating stuff.

Another option is to generate info-files in addition to the markdown. Create a new `examples-to-info.el` and go wild. An info-page wouldn't need the same exact content as is in the skeleton-file either, so making another one custom for info-pages sounds good too.

**Comment by holomorph** [comment]
There does not appear to be any org analog to texinfo things like, for instance: `@table` for explaining commands (e.g. in the Installation section) and `@defun` for API function documentation, so without embedding texinfo into org (which will make github rendering fail miserably), org mode will not work.

I'm going to look into writing/generating texinfo instead.

**Comment by holomorph** [comment]
Some generation and a little bit of poking later, here is an example of an info manual. You can load this into a Texinfo buffer and to `C-c RET C-b` to view it: https://gist.github.com/holomorph/3ffaa733a3ff02af4c96

Some things I noticed:
* in texinfo `@`, `{`, and `}` need to be escaped and there are two places where that's a problem: the `,@` operator and an example which has braces in it. Haven't quite figured out where to do the escaping
* some of the example docstrings have markdown quotes in them where they probably should be quoted differently
* Having a hard time making a node list like
```
* Maps::
* Sublist selection::
* List to list::
* Reductions::
* Unfolding::
* Predicates::
* Partitioning::
* Indexing::
* Set operations::
* Other list operations::
* Tree operations::
* Threading macros::
* Binding::
* Destructive operations::
* Function combinators::
```
from the function list

**Comment by Fuco1** [comment]
Fixed via #120.
